### PR TITLE
start with settings and connection-manager plugins

### PR DESCRIPTION
**Pull request by bucko13**
Simple update to start new installs with the connection manager and settings (especially good for setting up new environments).

### CODE REVIEW

File: scripts/preinstall.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const CLIENTS_DIR = path.resolve(BPANEL_DIR, 'clients');
 const LOCAL_PLUGINS_DIR = path.resolve(BPANEL_DIR, 'local_plugins');
 
 const configText = `module.exports = {
-  plugins: ['@bpanel/genesis-theme', '@bpanel/bui'],
+  plugins: ['@bpanel/genesis-theme', '@bpanel/bui', '@bpanel/settings', '@bpanel/connection-manager'],
```
Comment (tynes) [inline scripts/preinstall.js]: What do you think about removing `bui` as a default plugin at this point? Now that we have some actual utility, it doesn't make sense to me as much to include it

Comment (pinheadmz) [reply scripts/preinstall.js]: Agree with removing `bui` -- should this change also be applied in `docker-compose.yml` ?

Comment (tynes) [reply scripts/preinstall.js]: I agree with removing it from `docker-compose` as well

Comment (bucko13) [reply scripts/preinstall.js]: Agreement all around! Will update accordingly.

**Pull request merged**